### PR TITLE
Added solution Football Match

### PR DESCRIPTION
**Pull request by nevskyy**

### CODE REVIEW

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 1]
```diff
@@ -15,6 +15,8 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        return (result1 == bet1 && result2 == bet2)
+                ? 2 : (result1 - result2) * (bet1 - bet2) > 0
+                ? 1 : (result1 - result2) == 0 && (bet1 - bet2) == 0 ? 1 : 0;
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FootballMatch.java]: don't use arithmetical operations (+ - / *) to compare numbers

**Comment by nevskyy** [comment]
Done

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 2]
```diff
@@ -15,6 +15,11 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        return ((result1 == bet1 && result2 == bet2) && (result1 > result2 && bet1 > bet2))
+                || ((result1 == bet1 && result2 == bet2) && (result1 < result2 && bet1 < bet2))
+                || ((result1 == bet1 && result2 == bet2) && (result1 == result2 && bet1 == bet2))
+                ? 2 : ((result2 > result1 && bet2 > bet1) || (result2 < result1 && bet2 < bet1)
+                || (result1 == result2 && bet1 == bet2)) ? 1 : 0;
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FootballMatch.java]: too much checks. try to simplify

**Comment by nevskyy** [comment]
Simplified

**Review by Tarabarnik — APPROVED** [review]

**Pull request closed without merging**